cve_bin_tool/output_engine/__init__.py

```diff
@@ -92,19 +89,53 @@ def output_pdf(all_cve_data: Dict[ProductInfo, CVEData], outfile):
                 product_info.vendor,
                 product_info.product,
                 product_info.version,
-                cve.cve_number,
-                cve.severity,
             ]
             pdfdoc.addrow(
                 "Vulnlist",
                 entry,
-                [
-                    ("TEXTCOLOR", (3, row), (4, row), severity_colour[cve.severity]),
-                    ("FONT", (3, row), (4, row), "Helvetica-Bold"),
-                ],
             )
             row += 1
-    pdfdoc.showtable("Vulnlist", widths=[3 * cm, 2 * cm, 2 * cm, 4 * cm, 3 * cm])
+    pdfdoc.showtable("Vulnlist", widths=[3 * cm, 2 * cm, 2 * cm])
+    pdfdoc.paragraph(
+        f"There are {products_with_cve} products with vulnerabilities found."
+    )
+    pdfdoc.pagebreak()
+    if products_with_cve != 0:
+        pdfdoc.heading(1, "List of Identified Vulnerabilities")
+        pdfdoc.paragraph(
+            "The following vulnerabilities are reported against the identified versions of the libraries."
+        )
+        pdfdoc.createtable(
+            "Vulnlist",
+            ["Vendor", "Product", "Version", "CVE Number", "Severity"],
+            pdfdoc.tblStyle,
+        )
+        row = 1
+        for product_info, cve_data in all_cve_data.items():
+            for cve in cve_data["cves"]:
+                if cve.cve_number != "UNKNOWN":
```

Comment:
If `"UNKNOWN"` is being skipped, might as well remove it from `severity_colour` dict.